### ISSUE TITLE
Bazel RBE Asan C/C++ broken

### ISSUE THREAD

**Issue by yashykt** [grpc/grpc#17846]
Seeing errors of the form

```
==18==LeakSanitizer has encountered a fatal error.
==18==HINT: For debugging, try setting environment variable LSAN_OPTIONS=verbosity=1:log_threads=1
==18==HINT: LeakSanitizer does not work under ptrace (strace, gdb, etc)
```

Sample log files from different pull requests -
https://source.cloud.google.com/results/invocations/f085bf47-8977-431e-9f95-4fef2c8f8301/targets
https://source.cloud.google.com/results/invocations/93a85600-029e-4538-b3fa-3b8ebac9b3f3/targets

**Comment by jtattermusch** [comment]
This will likely be fixed by #17815

**Comment by jtattermusch** [comment]
Seems fixed.
